vm/vm_test.go

```diff
@@ -395,7 +396,7 @@ func testMonitorExecution(t *testing.T, test *Test) {
 		test.BodyExecuting(testInst.outc, testInst.errc, inject)
 		done <- true
 	}()
-	_, rep, err := inst.Run(time.Second, reporter, "", opts...)
+	_, rep, err := inst.Run(context.Background(), reporter, "", opts...)
```

Comment:
Was the removal of `time.Second` intended here?

Reply:
Yes. It is anyway ignored in testInstance.Run.

Reply:
Let's use nil to make it more obvious.

Reply:
> Error: SA1012: do not pass a nil Context, even if a function permits it; pass context.TODO if you are unsure about which Context to use (staticcheck)

context.Background() looks the best from the static checker point of view